docs/book/src/component-config-tutorial/testdata/project/Makefile

```diff
@@ -141,15 +141,14 @@ ENVTEST ?= $(LOCALBIN)/setup-envtest
 KUSTOMIZE_VERSION ?= v5.0.1
 CONTROLLER_TOOLS_VERSION ?= v0.12.0
 
-KUSTOMIZE_INSTALL_SCRIPT ?= "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
```

Comment:
Can you please have your branch rebased with master and run `make generate` to update the samples and docs?
Also, please when you push the PR to update this one can you have the commits squashed?

Reply:
My apology, I forgot to setup `upstream` using `git remote`

Reply:
all fine. Thank you